audio/voices.yaml

```diff
@@ -1,28 +1,26 @@
-AlliedEngineerVoice:
-	DefaultVariant: .wav
-	Voices:
-		Select: ienasea, ienaseb, ienasec, ienased
-		Move: ienamoa, ienamob, ienamoc
-		Action: ienaata, ienaatb, ienaatc
-		Feedback: ienafea, ienafeb, ienafec
-		Die: ienadia, ienadib, ienadic, ienadid
+EngineerVoice:
+	DefaultVariant: .wav
+	Prefixes:
```

Comment:
> @scshunt @MustaphaTR wants such missing sounds remapped to what exists. That would do render such missing sounds harder to find out.

That's not what i want. I think we need a way to set counts seperately. I didn't mean game should play others if sound isn't found.